src/yki/handler/yki_register_debug.clj

```diff
@@ -23,4 +27,19 @@
         :path-params [id :- ::ys/id]
         (log/warn (str "Request yki-register CSV export for debug purposes for exam-session " id))
         (-> (ok (return-exam-session-participants-csv db url-helper id))
-            (assoc-in [:headers "Content-Type"] "text/csv"))))))
+            (assoc-in [:headers "Content-Type"] "text/csv")))
+      (context "/sync/exam-session" []
+        :coercion :spec
+        :no-doc true
+        :middleware [auth access-log wrap-params]
+        (POST "/:id" _
+          :path-params [id :- ::ys/id]
+          (log/info "Manually forcing exam session" id "to be synced to Solki")
+          (let [exam-session (exam-session-db/get-exam-session-by-id db id)]
+            (sync-exam-session-and-organizer db url-helper basic-auth false {:type         "CREATED"
```

Comment:
Jos ymmärrän tätä oikein, niin viimeisenä parametrina haluttais antaa organizer-oid, mutta `created` ei sellainen jota tuosta luettais?

Reply:
Tuon `sync-exam-session-and-organizer` signaturessa vika parametri on muotoa `{:keys [type exam-session organizer-oid]}`, ts. vikana parametrinä oletetaan olevan map muotoa `{:type "CREATED"/"DELETED", :exam-session ..., :organizer-oid ...}`

Reply:
Tuo `sync-exam-session-and-organizer` on kirjoitettu hieman sekavasti, koska sen tiedoston rivillä 187 bindataan `organizer-oid` uudelleen tuon `exam-session` sisältä tulevaan arvoon.

Reply:
-> Lopputuloksena kun tässä kutsutaan `sync-exam-session-and-organizer` siten että `type` on aina `"CREATED"`, funktion vikan parametriin ei tuota `organizer-oid` arvoa tartte sisällyttää, vaan se luetaan sitten `exam-session` avaimen alta.

Reply:
Ehkä tää on sen verran sekava että voisin yrittää refaktoroida tuota hieman.

Reply:
Joo. Mut eikös tässä pitäis siis antaa tuo `:organizer-oid` myös mapissa, kun taas `:created` keywordillä ei nähdäkseni tehdä mitään? Vai ymmärränkö väärin.

Reply:
Aivan joo, toi `:created ...` lienee turha. Se taisi tulla mukaan kohdista joissa synkronointikutsut kierrätetään persistoidun jonon kautta, jolloin itemin luontihetki on tärkeä mm. retry-logiikan kannalta.

Reply:
Mutta `:organizer-oid`ia ei tuossa tosiaan tarvitse antaa, koska vastaava arvo luetaan tuolta `:exam-session` sisältä.